### PR TITLE
model-conversion : add missing curl script [no ci]

### PR DESCRIPTION
**Pull request by danbev**
This commit adds a curl script to the model-conversion examples which is currently missing. This script is required for the running the embedding server targets to test llama-server embeddings functionality.

### CODE REVIEW

**Review by CISC — APPROVED** [review]

**Comment by CISC** [comment]
BTW, all the scripts should probably be updated to use `/usr/bin/env`, also noticed that some of them have incorrect file-endings, like `run-casual-gen-embeddings-org.sh`, which is a Python script. :)

Edit: Oh, and some even have missing hash-bangs: `compare-embeddings-logits.sh`, `create-collection-add-model.sh`...

**Comment by danbev** [comment]
@CISC Thanks, I'll go through them add fix these issues :+1:
